docs/typescript-errors.md

```
@@ -0,0 +1,180 @@
+**Line 30**: Cannot find name 'IntersectionObserver'
+**Line 31**: Parameter 'entries' implicitly has 'any' type
+
+#### src/ui/viewer/components/Sidebar.tsx (3 errors)
+**Lines 81, 99, 113**: Property 'value' does not exist on EventTarget
+
+---
+
+## Priority Fix Order
+
+1. **High Priority - Breaks build:**
+   - src/shared/config.ts (__DEFAULT_PACKAGE_VERSION__)
```

markdownlint-cli2 (0.18.1)

Strong style
Expected: asterisk; Actual: underscore

(MD050, strong-style)